### PR TITLE
fixed description for the cstg example website

### PR DESCRIPTION
**Pull request by sunnywu**
instead of Server-Side it is Client-Side integration and added a link to https://unifiedid.com/docs/guides/publisher-client-side guide

### CODE REVIEW

File: examples/cstg/html/index.html [hunk 1]
```diff
@@ -75,8 +75,14 @@
     </script>
   </head>
   <body>
-    <h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
-    <!-- TODO: Add introductory text here. -->
+    <h1>Client-Side Integration Example, UID2 JavaScript SDK</h1>
+    <p>
+      This example demonstrates how a content publisher can follow the 
+      <a href="https://unifiedid.com/docs/guides/publisher-client-side"
+      >Client-Side Integration Guide for JavaScript </a
```
Comment (alex-yau-ttd) [inline examples/cstg/html/index.html:82]: nit: can we please put the closing tag on the same line as the opening? i.e.
```
  >Client-Side Integration Guide for JavaScript
</a>
```

Comment (sunnywu) [reply examples/cstg/html/index.html]: ok i think i fixed as you requested?

**Review by genwhittTTD — APPROVED** [review]

File: examples/cstg/html/index.html (hunk 1 above)
Comment (alex-yau-ttd) [reply examples/cstg/html/index.html]: close, line 82 should be indented:
```
<a href="https://unifiedid.com/docs/guides/publisher-client-side"
  >Client-Side Integration Guide for JavaScript
</a>
```

Comment (alex-yau-ttd) [reply examples/cstg/html/index.html]: Auto-formatter in your IDE should hopefully fix this automatically?

**Review by alex-yau-ttd — APPROVED** [review]

File: examples/cstg/html/index.html (hunk 1 above)
Comment (sunnywu) [reply examples/cstg/html/index.html]: fixed as discussed offline

**Pull request merged**